com.tiny-beast-dictionary.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.Arian94.Tiny-beast-dictionary
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: tiny-beast-dict
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+    --socket=pulseaudio
+  - --filesystem=xdg-documents
+modules:
+  - name: tiny-beast-dictionary
+    buildsystem: simple
+    build-commands:
+      - install -D src-tauri/tiny-beast-dictionary /app/bin/tiny-beast-dictionary
+      - cp -r src-tauri/tiny-beast-dictionary.desktop /app/share/applications/io.github.Arian94.Tiny-beast-dictionary.desktop
+      - cp -r src-tauri/icons/32x32.png /app/share/icons/hicolor/32x32/apps/io.github.Arian94.Tiny-beast-dictionary.png
+      - cp -r src-tauri/icons/128x128.png /app/share/icons/hicolor/128x128/apps/io.github.Arian94.Tiny-beast-dictionary.png
+      - cp -r src-tauri/icons/256x256.png /app/share/icons/hicolor/256x256/apps/io.github.Arian94.Tiny-beast-dictionary.png
+      - cp -r src-tauri/xsel /app/bin/xsel
```

Comment:
this looks like a binary. How does that work with both `x86_64` and `aarch64`. Can't it be just built from source?